api/routes/user_routes.py

```diff
@@ -0,0 +1,67 @@
+from db.queries.assessment_records.queries import create_user_application_association
+from db.queries.assessment_records.queries import get_user_application_associations
+from db.queries.assessment_records.queries import (
+    update_user_application_association as update_user_application_association_db,
+)
+from db.schemas.schemas import AllocationAssociationSchema
+from flask import abort
+from flask import current_app
+from flask import request
+
+
+def get_all_users_associated_with_application(application_id, active=None):
+    associations = get_user_application_associations(application_id=application_id, active=active)
```

Comment:
Sure, added some comments. I don't think there's much value adding logging here for such simple routes, and it will likely just pollute the log store.